locations/Bahraich/inputs/scenario/scenario_inputs.yaml

```diff
@@ -17,7 +17,9 @@ scenarios:
       backup:
         threshold: 0.1 # Maximum acceptible blackouts (0.0-1.0)
     grid: true # "true" or "false"
-    grid_type: bahraich # The grid profile to use
```

Comment:
Again, this looks good for now, but, when we're done, we want something that looks kind of like the default files sitting in Bahraich which you can then specialise in your own location. This comes back to the comment about working in a location that isn't Bahraich so that the changes don't affect everyone's default location 😺